packages/indexer-nibi/src/heart-monitor.ts

```diff
@@ -128,8 +128,8 @@ export class HeartMonitor implements IHeartMonitor {
       this.gqlEndpt = this.defaultGqlEndpt
     } else if (typeof gqlEndpt === "string") {
       this.gqlEndpt = gqlEndpt
-    } else if (chain?.endptTm !== undefined) {
-      const endptFromRpc: string | null = gqlEndptFromTmRpc(chain?.endptTm)
+    } else if (chain && chain.endptTm !== undefined) {
+      const endptFromRpc: string | null = gqlEndptFromTmRpc(chain.endptTm)
```

Comment:
Trivially derived types? `string | null`

Reply:
Yes, gqlEndptFromTmRpc seems to return either type. But there is ternary operator for both type cases after

```
 } else if (chain && chain.endptTm !== undefined) {
      const endptFromRpc: string | null = gqlEndptFromTmRpc(chain.endptTm)
      this.gqlEndpt =
        endptFromRpc !== null ? endptFromRpc : this.defaultGqlEndpt
```

Reply:
Well, gqlEndptFromTmRpc already returns the types for endptFromRpc, so redefining the types here doesn't need to happen